### ISSUE TITLE
Training and Testing 

### ISSUE THREAD

**Issue by YasminAMassoud** [garethjns/Kaggle-EEG#17]
I was able to run the code, but i need to edit parts in it to enhance the results, 
I need to understand what files are used to produce training AUC , i mean are they trained against a hold set of the train files for example ? and same for testing is it compared to previously trained data? Can you point these parts in the code 

Thanks

**Comment by garethjns** [comment]
Hi @YasminAMassoud,

The AUC is calculated here https://github.com/garethjns/Kaggle-EEG/blob/c8883b1b1371b89781b2f82f412559ddbca5f362/%40seizureModel/seizureModel.m#L180 There's no hold out set in the training script, however the AUC calculated is the average over the test portions of the cross validation folds.

**Comment by YasminAMassoud** [comment]
the average over the test portions of the cross validation folds---> are these all the test files divided into 6 folds ? or part of the training files divided into test folds? ( I mean during prediction phase)

**Comment by Squirrel1489** [comment]
Is this still an active project? I ask because my two year old has had what was told to us last year focal epilepsy and now she is in the hospital with the shakes due to her muscles and entire nural network on the fritz. I'm looking to learn and work with someone or everyone to figure out whats wrong with my child.

**Comment by garethjns** [comment]
Hi @Squirrel1489 ,

Sorry to hear about your daughter and I hope everything turns out well.

I'm not actively working on this project anymore, however seizure prediction generally is still a very active research area. For more information see http://www.epilepsyecosystem.org and http://levinkuhlmann.byethost3.com/?i=1

@YasminAMassoud apologies, only just saw your question. The test folds in the cross-validation will be using data from the training dataset only. The data in the Kaggle test set wouldn't have been used in cross-validation.

**Comment by YasminAMassoud** [comment]
Hello @Squirrel1489 Hope you child becomes better and is feeling well. This is still an active research area i hope you find a current solution to her medical issue. 
@garethjns Thanks for your constant reply .
